packages/server/src/arpa_reporter/services/validate-data/validate.js

```diff
@@ -200,14 +200,21 @@ function isUnitedStates(value) {
     return value === 'usa' || value === 'united states';
 }
 
-function isValidState(val, content) {
+// TODO: re-write this
+// eslint-disable-next-line no-unused-vars
+function dropdownIncludes(key) {
+    // eslint-disable-next-line no-unused-vars
+    return (val) => true;
+}
```

Comment:
@NatHillardUSDR is it expected that this function is still a stub?

Reply:
Actually, this might pre-date you, huh.  I think your name showed up in git blame, but that could be just the eslint --fix operation from last week.

Reply:
Yeah, this wasn't my doing! This got blame impact is an interest side effect I didn't realize around linting!

Reply:
In fact, when I touched similar areas of this last, I realized that the entire `validate-data` codepath had been orphaned. 

At one time it was in use in validate-upload.js in arpa-reporter (prior to [8f1e12555dec8764241c90bf67325f81295a6aa7](https://github.com/usdigitalresponse/arpa-reporter/commit/8f1e12555dec8764241c90bf67325f81295a6aa7)) , but that commit seemed to remove the various validations done in those files. Some of this has been moved over to the Logic sheet but much of it hasn’t, it looks like.

In any case, I can file a separate issue for that problem, just wanted to note it here for reference